src/object/srv_obj.c

```diff
@@ -1913,9 +1913,8 @@ obj_local_enum(struct obj_io_context *ioc, crt_rpc_t *rpc,
 	 * is our epoch. (See dc_obj_shard_list.)
 	 */
 	rc = dtx_begin(ioc->ioc_coc, &oei->oei_dti, oei->oei_epr.epr_hi,
-		       oei->oei_flags & ORF_EPOCH_UNCERTAIN, oei->oei_map_ver,
-		       &oei->oei_oid, 0, DAOS_INTENT_DEFAULT, NULL, 0, NULL,
-		       &dth);
+		       oei->oei_flags & ORF_EPOCH_UNCERTAIN, 1,
```

Comment:
For obj_local_enum(), the parameter "sub_modification_cnt" should be zero, not "1".